vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -760,6 +770,20 @@ void LedgerImpl::GetExternalWallet(const std::string& wallet_type,
       return;
     }
 
+    if (wallet_type == constant::kWalletGemini) {
```

Comment:
these consts should also be replaced with an enum

Reply:
Follow-up issue for replacing const strings with enums - https://github.com/brave/brave-browser/issues/16815

Reply:
https://github.com/brave/brave-core/pull/8566#discussion_r665651272